components/expenses/ExpenseSummary.js

```diff
@@ -232,7 +237,7 @@ const ExpenseSummary = ({ expense, collective, host, isLoading, permissions, sho
                 <PayoutMethodTypeWithIcon type={expense.payoutMethod?.type} />
               </Box>
               <div data-cy="expense-summary-payout-method-data">
-                <PayoutMethodData payoutMethod={expense.payoutMethod} />
+                <PayoutMethodData payoutMethod={expense.payoutMethod} isLoading={isLoadingLoggedInUser} />
```

Comment:
We're loading the payoutMethod from the expense object, I believe this should be:
```suggestion
                <PayoutMethodData payoutMethod={expense.payoutMethod} isLoading={isLoading || isLoadingLoggedInUser} />
```

Reply:
Good catch, thank you!